parser/skip_valid_block.go

```diff
@@ -0,0 +1,16 @@
+package parser
+
+import "strings"
+
+var skipValidBlocks = `types
+map
+`
+
+var SkipValidBlocks map[string]struct{} = map[string]struct{}{}
```

Comment:
// SkipValidBlocks is a mapping used to store names of valid blocks that should be skipped.
// An empty struct is used as the value type to occupy minimal space, serving only as a presence indicator